CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.24-dev
+
+* Extend `deserializeTimeout`.
```

Comment:
DBC: This could be worded better. Changelog entries should generally be written to be consumed by end-users, who won't have any idea what `deserializeTimeout` refers to, especially when it's formatted like a Dart identifier. Something like "Time out more slowly when loading test suites" would be clearer and more user-focused.